ci-operator/config/openshift/operator-framework-olm/openshift-operator-framework-olm-master.yaml

```diff
@@ -28,6 +32,18 @@ images:
       as:
       - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16
   to: operator-registry
+- dockerfile_path: operator-framework-cli.Dockerfile
```

Comment:
I don't know if o-sdk could be included in a future iteration, but I wanted to leave the door open, so to speak.

Reply:
That was https://github.com/openshift/operator-framework-olm/pull/704

Reply:
out-of-bound we agreed to leave the door open for future o-f binaries to be added for v1, kubectl plugins, though we're also going to base on scratch rather than a self-serving image.